### DISCUSSION TITLE
Custom certs with Pocketbase as a framework

### DISCUSSION THREAD

**Discussion by n-ham** [pocketbase/pocketbase#2786] [Q&A]
I think I saw it mentioned that one can now use custom certs when using Pocketbase as a framework? How does one do this? Any plans to offer a way to specify certs when using `./pocketbase serve` too?

**Comment by ganigeorgiev** [comment]
There is no dedicated method for loading custom certs, but with v0.17.0 the `OnBeforeServe()` event will have new [`http.Server` and `autocert.Manager` fields](https://github.com/pocketbase/pocketbase/blob/39accdba58743e9c6306c45ef75781ceb4874b74/core/events.go#L75-L80) that can be used to adjust the net/http server behavior for more advanced use cases (including loading custom `http.Server.TLSConfig`).

Alternatively, you can put PocketBase behind a reverse proxy (eg. nginx, apache, caddy, etc.) and load the custom certificate from there.
